Makefile

```diff
@@ -40,73 +45,47 @@ install-clean:
 install-doc:
 	go install -gcflags "$(GC_FLAGS)" ./src/server/cmd/pachctl-doc
 
+doc-custom: install-doc install-clean
+	./etc/build/doc.sh
+
+doc:
+	@make VERSION_ADDITIONAL= doc-custom
+
 point-release:
-	@make VERSION_ADDITIONAL= release-helper
-	@make VERSION_ADDITIONAL= release-pachctl
+	@./etc/build/make_changelog.sh $(CHLOGFILE)
+	@VERSION_ADDITIONAL= ./etc/build/make_release.sh
 	@make doc
 	@echo "Release completed"
 
 # Run via 'make VERSION_ADDITIONAL=-rc2 release-candidate' to specify a version string
 release-candidate:
-	@if [ $${VERSION_ADDITIONAL:0:1} != - ]; then \
-	  echo "VERSION_ADDITIONAL must start with a '-'"; \
-	  exit 1; \
-	fi
+	@make custom-release
+
+custom-release:
+	echo "" > $(CHLOGFILE)
+	@VERSION_ADDITIONAL=$(VERSION_ADDITIONAL) ./etc/build/make_release.sh
```

Comment:
I thought about this a little more, and is it both `VERSION` and `VERSION_ADDITIONAL` that isn't getting passed through? If it's just VERSION, maybe all that's needed is to put `export VERSION` at the top?